knowledge_runtime/tests/test_config_resolver.py

```
@@ -0,0 +1,1102 @@
+            "embeddingConfig": {"dimensions": 1536},
+        }
+    model = MagicMock()
+    model.name = model_name
+    model.namespace = model_namespace
+    model.kind = "Model"
+    model.json = {"spec": spec}
+    return model
+
+
+def _make_document(
+    document_id: int = 100, splitter_config: dict | None = object()
```

Ruff (0.15.11)

[warning] Do not perform function call `object` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)